src/components/form_elements.js

```diff
@@ -87,12 +87,12 @@ export function renderDateTimePicker({ input, label, required, meta: { touched,
   const inputProps = {
     disabled: disabled
   } 
-
+  
   return (
     <Form.Group as={Col} xs={xs} sm={sm} md={md} lg={lg}>
       <Form.Label>{label}{requiredField}</Form.Label>
       <Datetime className="rdtPicker-sealog" {...input} utc={true} value={input.value ? moment.utc(input.value).format(dateFormat + ' ' + timeFormat) : null} dateFormat={dateFormat} timeFormat={timeFormat} selected={input.value ? moment.utc(input.value) : null } inputProps={inputProps} />
-      {touched && (error && <div className="w-100 mt-1 text-danger" style={{fontSize: ".7rem"}}>{error}</div>)}
+      { (error && <div className="w-100 mt-1 text-danger" style={{fontSize: ".7rem"}}>{error}</div>) }
```

Comment:
Probably also should have removed the unnecessary parentheses (why were they there? who knows) and the extra whitespace, but no need to go back and fix it.